bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/internal/loader/ScriptFileReference.java

```diff
@@ -116,4 +113,21 @@ public int compareTo(ScriptFileReference other) {
             return 0;
         }
     }
+
+    @Override
+    @NonNullByDefault({})
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
```

Comment:
Please add brackets:

```suggestion
        if (this == o) {
            return true;
        }
```

Reply:
done